src/operator/src/statement/ddl.rs

```diff
@@ -477,7 +477,7 @@ fn create_table_info(
         next_column_id: column_schemas.len() as u32,
         region_numbers: vec![],
         options: table_options,
-        created_on: DateTime::default(),
+        created_on: Utc::now(),
```

Comment:
I don't know if we have better created time, but this can serve as the single source of truth.

BTW, it seems quite internal to write a whitebox UT. How can I write a blackbox UT to prevent regression, or just need not as trivial as it is.

Reply:
I think it's fine to skip adding a ut for it now, considering the "created_on" is not used anywhere seriously.

Reply:
I believe this bug has existed for at least half a year.🤣